File: #1-25 (C, Java, Python)/Python/Euler1.py
#Euler1 ~ Multiples of 3 and 5
import timeit
start = timeit.default_timer()

sum = 0;
for x in range(0,1000):
    if (x%3 == 0) or (x%5 == 0):
        sum = sum + x;

stop = timeit.default_timer()
print("Solution: ",sum,"\n")
print("Program completed in","{0:.3f}".format(stop - start), "seconds.")

File: #1-25 (C, Java, Python)/Java/Euler1.java
//Euler1 ~ Multiples of 3 and 5
import java.util.concurrent.TimeUnit;

public class Euler1
{
    public static void main(String[] args) throws InterruptedException
	{
		long startTime = System.nanoTime();
		
		int sum = 0;
		for(int i = 0; i < 1000; i++)
		{
			if(((i%3)==0) || ((i%5)==0))
			{
				sum = sum + i;
			}
		}
		
		long endTime = System.nanoTime();
		long timeElapsed = endTime - startTime;
		System.out.println("Solution: " + sum + "\n");
		System.out.println("Program completed in " + timeElapsed/1000000000.0 + " seconds.");
    }
}